lib/Reader/reader.cpp

```diff
@@ -6410,6 +6410,11 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
     CurrInstrOffset = CurrentOffset;
     NextInstrOffset = NextOffset;
 
+    // Set debug locations
+    if (ReaderOperandStack->empty()) {
+      setDebugLocation(CurrentOffset, 0);
```

Comment:
Agree with Pat that that second arg needs to convey information at call sites. If you make it a bool then change this to 
```
const bool IsCall = false;
setDebugLocation(CurentOffset, IsCall);
```
or similar.